.github/workflows/title.yml

```
@@ -0,0 +1,36 @@
+name: "Lint PR title"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+permissions:
+  pull-requests: read
+
+jobs:
+  main:
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@v5
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          types: |
+            BREAKING
+            [unstable] BREAKING
+            deprecation
+            [unstable] deprecation
```

```suggestion
            deprecation
```
dont think unstable is necessary/useful for JSR